src/app/space/app-launcher/app-launcher.module.ts

```diff
@@ -1,23 +1,18 @@
 import { CommonModule } from '@angular/common';
-import { HttpClient } from '@angular/common/http';
 import { NgModule } from '@angular/core';
 import {
   AuthHelperService,
   Config,
-  DependencyEditorTokenProvider,
   HelperService,
-  TokenProvider,
-  URLProvider
+  TokenProvider
 } from 'ngx-launcher';
 import { AUTH_API_URL, AuthenticationService } from 'ngx-login-client';
-import { ApiLocatorService } from '../../shared/api-locator.service';
 import { AppLauncherRoutingModule } from './app-launcher-routing.module';
 import { AppLauncherComponent } from './app-launcher.component';
 import { CreateAppModule } from './create-app/create-app.module';
 import { ImportAppModule } from './import-app/import-app.module';
 import { AuthAPIProvider } from './services/app-launcher-authprovider.service';
```

Comment:
Same as above. This is just a wrapper for getting auth api url. We already provide `AUTH_API_URL` in root module. Can be used directly.